frontend-opal/src/components/WatchList/WatchList/WatchList.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { View, Text } from 'react-native';
+import { AntDesign } from 'react-native-vector-icons';
+import { Button } from 'native-base';
+import PropTypes from 'prop-types';
+import styles from './styles';
+import InstrumentItems from '../InstrumentItems/instrumentItems';
+import ChartDetail from '../ChartDetail/ChartDetail';
+
+
+const WatchList = ({
+  watchlistArr, info, stockData, chartData,
+}) => (
+  <View>
+    {watchlistArr.map((item) => {
+      const [showwatchlist, setShowwatchlist] = useState(false);
+
+      const toggle = (parm) => !parm;
+
+      const changeShowWatchList = () => (
+        setShowwatchlist((toggle(showwatchlist)))
+      );
+      return (
+        <View key={item}>
+          <View style={styles.border}>
+            <Text>{item}</Text>
+            <Button onPress={changeShowWatchList}><AntDesign name="caretdown" size={15} color="black" /></Button>
+          </View>
+          {showwatchlist ? (
```

Comment:
@martinupset 
```suggestion
          {showwatchlist && (
```
and then you don't need the `: null` at the end

Reply:
ok